src/main/java/org/folio/rest/impl/PurchaseOrdersAPI.java

```diff
@@ -283,4 +283,43 @@ public void putOrdersStoragePurchaseOrdersById(String id, String lang, org.folio
       }
     });
   }
+
+  private void processCreateSequence(PurchaseOrder entity, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext,
+                                     String tenantId, String persistenceId, OutStream stream) {
+    PostgresClient.getInstance(vertxContext.owner(), tenantId).execute(CREATE_SEQUENCE.getQuery(entity.getId()),
+      createSequenceReply -> {
+        if(createSequenceReply.succeeded()) {
+          Response response = PostOrdersStoragePurchaseOrdersResponse
+            .respond201WithApplicationJson(stream, PostOrdersStoragePurchaseOrdersResponse.headersFor201()
+              .withLocation(PURCHASE_ORDER_LOCATION_PREFIX + persistenceId));
+          respond(asyncResultHandler, response);
+        } else {
+          // TODO Transactionality (???) --> Roll-back PO creation?
```

Comment:
I'd say leave the TODO until we can figure out a better way to handle these failures